src/scenes/Partners/Detail/PartnerDetail.graphql

```diff
@@ -57,12 +60,18 @@ fragment PartnerDetails on Partner {
     canEdit
     value
   }
-  projects {
-    canRead
-    canCreate
-    items {
-      ...ProjectListItem
-    }
+  projects(input: $ProjectListInput) {
```

Comment:
I believe it is needed in the fragment for gql generation. In any case without matching the query with the var like this we get:

`Error 0: Fields "projects" conflict because they have differing arguments. Use different aliases on the fields to fetch both if this was intentional.`

Reply:
Ah I see, so the shared fragment is actually using forwarding the args to the operation, idk if that's new or not but that makes sense.
This would go away with the new specific `PartnerProjects` query